xpublish_edr/formats/to_geojson.py

```diff
@@ -0,0 +1,33 @@
+"""
+Generate GeoJSON responses for an xarray dataset for EDR queries
+"""
+
+import geopandas as gpd
+import xarray as xr
+from fastapi import Response
+
+
+def to_geojson(ds: xr.Dataset):
+    """Return a GeoJSON response from an xarray dataset"""
+    ds = ds.squeeze()
+    x_col = ds.cf["X"].name
+    y_col = ds.cf["Y"].name
+    if "T" in ds.cf:
+        time_col = ds.cf["T"].name
+    else:
+        time_col = None
+
+    df = ds.to_dataframe().reset_index()
+    gdf = gpd.GeoDataFrame(df, geometry=gpd.points_from_xy(df[x_col], df[y_col]))
+
+    # Map the time to a string if applicable
+    # TODO: Handle timezone?
```

Comment:
xarray strips timezone info sadly so the dataframe here has timezone-naive timestamps :/

Reply:
Yeaaa which is the same as the CSV and others prints out for now, so ill leave it consistent for now